voluncheer/profiles/views/profile.py

```diff
@@ -1,11 +1,14 @@
 from django.contrib.auth.decorators import login_required
```

Comment:
Please keep imports sorted alphabetically. The exception is to keep external imports (one's we are not writing) separate from internal imports (the modules we are writing).